packages/framework/attributor/src/mixinAttributor.ts

```diff
@@ -0,0 +1,216 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { IDocumentMessage, ISequencedDocumentMessage, ISnapshotTree } from "@fluidframework/protocol-definitions";
+import { IAudience, IContainerContext, IDeltaManager } from "@fluidframework/container-definitions";
+import { ContainerRuntime } from "@fluidframework/container-runtime";
+import type { IContainerRuntimeOptions } from "@fluidframework/container-runtime";
+import { ISummaryTreeWithStats, ITelemetryContext, NamedFluidDataStoreRegistryEntries } from "@fluidframework/runtime-definitions";
+import { addSummarizeResultToSummary, SummaryTreeBuilder } from "@fluidframework/runtime-utils";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
+import { IRequest, IResponse, FluidObject } from "@fluidframework/core-interfaces";
+import { assert, bufferToString, unreachableCase } from "@fluidframework/common-utils";
+import { UsageError } from "@fluidframework/container-utils";
+import { AttributionInfo, AttributionKey, Attributor, IAttributor, OpStreamAttributor } from "./attributor";
+import { AttributorSerializer, chain, deltaEncoder, Encoder } from "./encoders";
+import { makeLZ4Encoder } from "./lz4Encoder";
+
+// Summary tree keys
+const attributorKey = "attributor";
```

Comment:
it seems like all the other system trees and blobs at this layer start with `.`

Reply:
also, it might be better to call it attributorTreeName

Reply:
https://github.com/microsoft/FluidFramework/blob/main/packages/runtime/container-runtime/src/summaryFormat.ts#L154L159